src/main/kotlin/org/rust/lang/core/psi/ext/RsDocAndAttributeOwner.kt

```diff
@@ -105,6 +105,11 @@ class QueryAttributes(
         return attrs.any()
     }
 
+    fun hasAttribute(attributeNameRe: Regex): Boolean {
```

Comment:
```suggestion
    fun hasAttribute(regex: Regex): Boolean {
```

Reply:
Ha, this parameter name was originally suggested by you xD https://github.com/intellij-rust/intellij-rust/pull/4758#discussion_r360304980

Reply:
Oh, it's indeed funny 😆 
But as they say two hardest things in programming are cache invalidation and naming things.
In this case, it makes sense because now we work with attribute text instead of its name. When we improve attribute parsing, I'll rename it in a proper way